tests/utils/faabric_utils.h

```diff
@@ -2,7 +2,7 @@
 
 #include <catch2/catch.hpp>
 
-#include "fixtures.h"
```

Comment:
Throughout the tests we included "faabric_utils.h" to use the test fixtures. However, the fixtures are defined in "fixtures.h", and it only workerd because we had this include here, which we actually don't need for "faabric_utils.h".